osquery/core/json.h

```diff
@@ -68,7 +69,22 @@ class JSON : private only_movable {
   rapidjson::Document getArray() const;
 
   /// Add a JSON object or array to a list.
-  void push(rapidjson::Document& line);
+  void push(rapidjson::Value& value);
+
+  /// Add a JSON object or array to a list.
+  void push(rapidjson::Value& value, rapidjson::Value& arr);
+
+  /// Add a size_t to a JSON array.
+  void push(size_t value);
+
+  /// Add a size_t to a JSON array.
+  void push(size_t value, rapidjson::Value& arr);
+
+  /// Add a size_t to a JSON array.
```

Comment:
nit: your comments here don't seem to follow what the functions are doing, mind double-checking these?